evictionfree/tests/test_schema.py

```diff
@@ -224,6 +224,17 @@ def test_it_raises_err_when_used_on_wrong_site(self):
             "This form can only be used from the Eviction Free NY site."
         )
 
+    def test_it_shows_discontinued_message(self, use_evictionfree_site):
+        self.create_landlord_details()
+        OnboardingInfoFactory(user=self.user)
+        HardshipDeclarationDetailsFactory(user=self.user)
+
+        with freezegun.freeze_time("2021-01-26"):
+            assert self.execute()["errors"] == one_field_err(
+                "This tool has been discontinued! Please reload the page for more details."
+            )
+
+    @pytest.mark.skip(reason="The tool has been discontinued")
```

Comment:
I decided to just skip this test instead of deleting it to make it easy to go back to the original functionality if we ever need to.